src/xpk/core/system_characteristics.py

```diff
@@ -136,6 +136,15 @@ def get_system_characteristics_by_device_type(
         AcceleratorType['GPU'],
         'a100-40gb-8',
     ),
+    'h200-141gb-8g': SystemCharacteristics(
+        'N/A',
+        1,
+        'h200-141gb-8g',
+        'a3-ultra-8g',
+        8,
+        AcceleratorType['GPU'],
+        'h200-141gb-8g',
```

Comment:
and from here, too: `h200-141gb-8`
make sure to update config_map if when you remove this.